src/test/java/org/broadinstitute/hellbender/utils/nio/ExtremeReadsTest.java

```diff
@@ -0,0 +1,95 @@
+package org.broadinstitute.hellbender.utils.nio;
+
+import com.google.common.base.Stopwatch;
+import java.nio.ByteBuffer;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+/**
+ * Stress test for reading lots of data from the cloud using a very small prefetch buffer.
+ * Do not run this too often.
+ */
+public final class ExtremeReadsTest extends BaseTest implements Runnable {
```

Comment:
Can you implement a static inner `Runnable` class, instead of making the test class itself `Runnable`?

Reply:
Done.